components/icons/Incorrect.tsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable max-len */
```

Comment:
Why disable `max-len` this file? Consider only disable where it is needed.

Reply:
Agree